installation/routines/setup_autohotspot_NetworkManager.sh

```diff
@@ -0,0 +1,103 @@
+
+
+AUTOHOTSPOT_TIMER="autohotspot.timer"
+AUTOHOTSPOT_TIMER_PATH="${SYSTEMD_PATH}/${AUTOHOTSPOT_TIMER}"
+AUTOHOTSPOT_NETWORKMANAGER_RESOURCES_PATH="${INSTALLATION_PATH}/resources/autohotspot/NetworkManager"
+
+_install_packages_NetworkManager() {
+    sudo apt-get -y install iw
+}
+
+_install_autohotspot_NetworkManager() {
+    # configure interface conf
+    _config_file_backup "${AUTOHOTSPOT_INTERFACES_CONF_FILE}"
+    sudo rm "${AUTOHOTSPOT_INTERFACES_CONF_FILE}"
+    sudo touch "${AUTOHOTSPOT_INTERFACES_CONF_FILE}"
+
+    # create service to trigger hotspot
+    local ip_without_last_segment=$(_get_last_ip_segment $AUTOHOTSPOT_IP)
+    sudo cp "${AUTOHOTSPOT_NETWORKMANAGER_RESOURCES_PATH}"/autohotspot "${AUTOHOTSPOT_TARGET_PATH}"
+    sudo sed -i "s|%%WIFI_INTERFACE%%|${WIFI_INTERFACE}|g" "${AUTOHOTSPOT_TARGET_PATH}"
+    sudo sed -i "s|%%AUTOHOTSPOT_SSID%%|${AUTOHOTSPOT_SSID}|g" "${AUTOHOTSPOT_TARGET_PATH}"
+    sudo sed -i "s|%%AUTOHOTSPOT_PASSWORD%%|${AUTOHOTSPOT_PASSWORD}|g" "${AUTOHOTSPOT_TARGET_PATH}"
+    sudo sed -i "s|%%AUTOHOTSPOT_IP%%|${AUTOHOTSPOT_IP}|g" "${AUTOHOTSPOT_TARGET_PATH}"
+    sudo sed -i "s|%%IP_WITHOUT_LAST_SEGMENT%%|${ip_without_last_segment}|g" "${AUTOHOTSPOT_TARGET_PATH}"
+    sudo sed -i "s|%%AUTOHOTSPOT_TIMER_NAME%%|${AUTOHOTSPOT_TIMER}|g" "${AUTOHOTSPOT_TARGET_PATH}"
+    
+    sudo chmod +x "${AUTOHOTSPOT_TARGET_PATH}"
+
+    sudo cp "${AUTOHOTSPOT_NETWORKMANAGER_RESOURCES_PATH}"/autohotspot.service "${AUTOHOTSPOT_SERVICE_PATH}"
+    sudo sed -i "s|%%AUTOHOTSPOT_SCRIPT%%|${AUTOHOTSPOT_TARGET_PATH}|g" "${AUTOHOTSPOT_SERVICE_PATH}"
+    #sudo chown root:root "${AUTOHOTSPOT_SERVICE_PATH}"
```

Comment:
Is it intentional that it is commented out?

Reply:
yep, still a draft and some testing to do ;)